internal/ingress/annotations/mirror/main.go

```diff
@@ -85,5 +90,18 @@ func (a mirror) Parse(ing *networking.Ingress) (interface{}, error) {
 		config.Source = ""
 	}
 
+	config.Host, err = parser.GetStringAnnotation("mirror-host", ing)
```

Comment:
What happens if config.Host is an invalid Host? Like _sbrubles-@xpto:12345

Is there any implication on that? Just asking, we need to make sure this doesn't leaks or generates panics :)

Reply:
Hello

Thank you guys for response!

>add docs
Yeah I have added it.

>What happens if config.Host is an invalid Host? Like _sbrubles-@xpto:12345

It's not a problem, because as I understand right, host part of URI have no strict rules. 

Just for example - this string ` _sbrubles-i"@xpto:12345` generates this nginx config (I have added quotes for test):

location = /_mirror-1743e3cd-cd3a-40dd-8594-a377b6700200 {
			internal;
			proxy_set_header Host _sbrubles-i"@xpto:12345;
			proxy_pass http://test.env.com/somepath$request_uri;
		}

It's passing nginx config test and don't gather any panic in ingress-nginx.

Reply:
Can you add a test case to cover this one?